contrib/neovim/template.py

```diff
@@ -28,7 +28,8 @@
 depends = [
     "lua5.1-lpeg",
     "tree-sitter-lua",
-    "tree-sitter-query",
+    "tree-sitter-markdown",
+    "tree-sitter-query<0.4.0",
```

Comment:
```suggestion
    "tree-sitter-query",
```

don't need explicit versions

Reply:
right. forgot to remove this after adding `ignore = True`.

removed :+1: